packages/contracts/contracts/registryManager/RegistryManager.sol

```diff
@@ -57,17 +57,20 @@ contract RegistryManager is Ownable, IRegistryManager, ICommon {
   /// @inheritdoc IRegistryManager
   function process(Request memory request) public virtual override isValidRequest(request) {
     request.status = Status.Pending;
-    requests[requestCount] = request;
+    uint256 index = requestCount;
```

Comment:
what does the `index` mean? 
I found that the `index` was renamed to `recipientIndex` and assigned as `request.index`, and the `index` was added. So what's the difference between `index` and `recipientIndex` (`request.index`) ?

Reply:
`recipientIndex` - is index inside `recipients` array and `index` is for `requests`